### PR TITLE
feat(Quick Profile): Implement Quick Profile

### PR DESCRIPTION
**Pull request by genie-magic**
**What this PR does** 📖
When you click a users picture in chat, and the quick profile appears, it should have their name/icon, and the ability to message them
**Which issue(s) this PR fixes** 🔨

Fixes # AP-268

**Special notes for reviewers** 🗒️
I think all functionalities are implemented except icon.
I can't test user icon on quick profile because.. updating icon on settings/profile is not fully implemented.
We can create individual ticket for that.

### CODE REVIEW

**Review by stavares843 — CHANGES_REQUESTED** [review]
when you click on your own avatar shows the option to message yourself

Captura de ecrã 2021-12-15, às 17 38 14

**Review by stavares843 — CHANGES_REQUESTED** [review]
Captura de ecrã 2021-12-15, às 17 40 16

when you go to the quick profile of a user > message > it doesn't have the 256 chars limit

**Review by stavares843 — CHANGES_REQUESTED** [review]
Captura de ecrã 2021-12-15, às 17 41 49

on mobile appears like this when trying to access the quick profile

iPhone X

File: components/interactables/QuickProfile/QuickProfile.html [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+  />
   <TypographyText :text="user.status" :size="6" />
   <UiHorizontalRule />
-  <InteractablesInputGroup :icon="{ style: 'far', name: 'arrow-circle-right' }" type="primary" :placeholder="`Message ${user.name}...`" />
-</div>
\ No newline at end of file
+  <InteractablesInputGroup
+    type="primary"
+    v-model="text"
+    :placeholder="`Message ${user.name}...`"
```
Comment (WanderingHogan) [inline components/interactables/QuickProfile/QuickProfile.html:21]: can you add 'Message' to the translations

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
looks good when Sara's changes go in

**Comment by genie-magic** [comment]
> 

@stavares843  you see that screen because you just inspect element and change it to mobile... please f5 and try from fresh mobile mode.. 
image

**Comment by genie-magic** [comment]
> looks good when Sara's changes go in

@WanderingHogan , @stavares843  I fixed issues you mentioned and pushed. only text limit is not implemented because Nick is working on there and I want to avoid conflict. After he implemented I can re work.

**Comment by stavares843** [comment]
when I refreshed stayed the same

will check again and check the fixes

**Review by WanderingHogan — APPROVED** [review]

**Comment by stavares843** [comment]
neat! 🎉

**Review by stavares843 — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**